start-livia.sh

```
@@ -20,7 +20,7 @@ fi
 
 # Carrega variáveis de ambiente
 echo "📋 Carregando variáveis de ambiente..."
-export $(cat .env | xargs)
+export $(cat .env | grep -v '^#' | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)